server/models/ipfspins.ts

```diff
@@ -0,0 +1,40 @@
+import * as Sequelize from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type IpfsPinsAttributes = {
+  id: number;
+  address_id: number;
+  ipfs_hash: string;
+  created_at: Date;
+  updated_at: Date;
+};
+
+export type IpfsPinsInstance = ModelInstance<IpfsPinsAttributes>;
+
+export type IpfsPinsModelStatic = ModelStatic<IpfsPinsInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): IpfsPinsModelStatic => {
+  const IpfsPins = <IpfsPinsModelStatic>sequelize.define(
+    'IpfsPins',
+    {
+      id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true },
+      address_id: { type: dataTypes.INTEGER, allowNull: false },
+      ipfs_hash: { type: dataTypes.STRING, allowNull: false },
+    },
+    {
+      tableName: 'IpfsPins',
+      timestamps: true,
+      underscored: true,
+      createdAt: 'created_at',
```

Comment:
We should explicitly declare created_at/updated_at as rows in object above.

Reply:
Done

Reply:
Not fully fixed -- need to add them to the sequelize.define, not just the attributes.

Reply:
Done